hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenAirdropTest.java

```diff
@@ -2183,6 +2166,44 @@ final Stream<DynamicTest> airdropTo0x0Address() {
                     .payingWith(OWNER)
                     .hasKnownStatus(INVALID_ACCOUNT_ID));
         }
+
+        @HapiTest
+        @DisplayName("pending airdrop 1 token to more than 10 accounts")
+        final Stream<DynamicTest> pendingAirdropOneTokenToMoreThan10Accounts() {
+            final var accountNames = generateAccountNames(10);
+            return hapiTest(flattened(
+                    // create 10 accounts with 0 auto associations
+                    createAccounts(accountNames, 0),
+                    tokenAirdrop(distributeTokens(FUNGIBLE_TOKEN, OWNER, accountNames))
+                            .payingWith(OWNER)
+                            .hasKnownStatus(TOKEN_TRANSFER_LIST_SIZE_LIMIT_EXCEEDED)));
+        }
```

Comment:
We have a fungible token test here, and happen to have an NFT test in `TokenAirdropHandlerTest` (via calling the `transactionBodyAboveMaxTransferLimit()` method). It would be good to call out that we're testing NFT coverage specifically in `TokenAirdropHandlerTest`.

Reply:
The unit test initially was design to test both FT and NFT, since there was one common validation of the size of the transfers. It was modified to pass and to work at least with NFTs.
I added a second HAPI test for NFT airdrop here too.